### ISSUE TITLE
Dir segment: wrong colors for first directory in PhpStorm

### ISSUE THREAD

**Issue by julienfalque** [romkatv/powerlevel10k#1100]
I have a weird issue with the `dir` segment. I have configured it to show all directories with the same unified colors (light grey text on dark grey background). When using Terminator this works just fine:

![Capture d’écran de 2020-11-11 11-21-10](https://user-images.githubusercontent.com/1736542/98799931-0b254280-2410-11eb-9aef-a9940b695060.png)

But when using PhpStorm's terminal, the first directory always appears as white text on black background:

![Capture d’écran de 2020-11-11 11-10-38](https://user-images.githubusercontent.com/1736542/98799840-ed57dd80-240f-11eb-865c-d32653e4450e.png)

The issue also occurs when in a non-writable directory or if the first directory is `~`.

I've tried to change many settings with no luck. Here are the relevant settings in my `~/.p10k.zsh` file:

```
  typeset -g POWERLEVEL9K_DIR_BACKGROUND=236
  typeset -g POWERLEVEL9K_DIR_FOREGROUND=250
  typeset -g POWERLEVEL9K_SHORTEN_STRATEGY=truncate_to_unique
  typeset -g POWERLEVEL9K_SHORTEN_DELIMITER=…
  typeset -g POWERLEVEL9K_DIR_SHORTENED_FOREGROUND=250
  typeset -g POWERLEVEL9K_DIR_ANCHOR_FOREGROUND=250
  typeset -g POWERLEVEL9K_DIR_ANCHOR_BOLD=false
  local anchor_files=(
    .bzr
    .citc
    .git
    .hg
    .node-version
    .python-version
    .go-version
    .ruby-version
    .lua-version
    .java-version
    .perl-version
    .php-version
    .tool-version
    .shorten_folder_marker
    .svn
    .terraform
    CVS
    Cargo.toml
    composer.json
    go.mod
    package.json
    stack.yaml
  )
  typeset -g POWERLEVEL9K_SHORTEN_FOLDER_MARKER="(${(j:|:)anchor_files})"
  typeset -g POWERLEVEL9K_DIR_TRUNCATE_BEFORE_MARKER=false
  typeset -g POWERLEVEL9K_SHORTEN_DIR_LENGTH=1
  typeset -g POWERLEVEL9K_DIR_MAX_LENGTH=
  typeset -g POWERLEVEL9K_DIR_MIN_COMMAND_COLUMNS=40
  typeset -g POWERLEVEL9K_DIR_MIN_COMMAND_COLUMNS_PCT=50
  typeset -g POWERLEVEL9K_DIR_HYPERLINK=true
  typeset -g POWERLEVEL9K_DIR_SHOW_WRITABLE=v2
  typeset -g POWERLEVEL9K_DIR_DEFAULT_NOT_WRITABLE_FOREGROUND=255
  typeset -g POWERLEVEL9K_DIR_DEFAULT_NOT_WRITABLE_SHORTENED_FOREGROUND=255
  typeset -g POWERLEVEL9K_DIR_DEFAULT_NOT_WRITABLE_ANCHOR_FOREGROUND=255
  typeset -g POWERLEVEL9K_DIR_DEFAULT_NOT_WRITABLE_BACKGROUND=red
```

**Comment by romkatv** [comment]
Works fine in GNOME Terminal and Konsole, too. Probably a bug in PhpStorm's terminal, so it should be reported there instead.

**Comment by romkatv** [comment]
Closing. Please reopen if there is any indication that there is a bug in Powerlevel10k rather than in PhpStorm's terminal.

**Comment by julienfalque** [comment]
Reported here: https://youtrack.jetbrains.com/issue/WI-56914.

**Comment by romkatv** [comment]
If you provide complete instructions how to reproduce this issue and mention that it works in N other terminals (post screenshots of all of them), it'll drastically increase the chance of anything happening. If you make the instructions minimal (the smallest possible `~/.zshrc` to reproduce the issue with no `~/.p10k.zsh`), it'll increase the chances even higher.

**Comment by romkatv** [comment]
Keep in mind that the person on the receiving end of https://youtrack.jetbrains.com/issue/WI-56914 likely doesn't use Powerlevel10k or even zsh.

**Comment by julienfalque** [comment]
I should have done that from the beginning: while making my `~/.zshrc` file as small as possible, I noticed that disabling `POWERLEVEL9K_DIR_HYPERLINK` fixes the issue on PhpStorm.

This means there are actually two issues:
* PhpStorm supports hyperlinks but overrides their style;
* PhpStorm only considers the first directory as a hyperlink, not the whole path.

I tested in Gnome Terminal and hyperlinks work as expected and with the correct colors, so I assume both issues are on PhpStorm side.

I also updated my original post with a small `.zshrc` file that reproduces the issue. Thanks for your time!
